rainmaker_orchestrator/Dockerfile

```
@@ -1,19 +1,14 @@
-    g++ \
-    curl \
-    && rm -rf /var/lib/apt/lists/*
+COPY requirements.txt requirements.txt
+RUN pip install --no-cache-dir -r requirements.txt
+
+COPY ./src /app/src
 
-# Copy requirements and install Python packages
-COPY rainmaker_orchestrator/requirements.txt .
-RUN pip wheel --no-cache-dir --no-deps --wheel-dir /wheels -r requirements.txt
+ feat/integrate-alibaba-access-analyzer-12183567303830527494
```

Hadolint (2.14.0)

[error] unexpected 'f'
expecting '#', '\', ADD, ARG, CMD, COPY, ENTRYPOINT, ENV, EXPOSE, FROM, HEALTHCHECK, LABEL, MAINTAINER, ONBUILD, RUN, SHELL, STOPSIGNAL, USER, VOLUME, WORKDIR, a pragma, at least one space, or end of input

(DL1000)